README.md

```diff
@@ -18,6 +18,15 @@ of the pipeline description, and more.
  * **Mailing List**: image-builder@redhat.com
  * **Changelog**: <https://github.com/osbuild/osbuild/releases>
 
+### Principles
+
+1. OSBuild stages are never broken, only deprecated. The same manifest should always produce the same output.
+2. OSBuild stages should be explicit whenever possible instead of e.g. relying on the state of the tree.
+3. OSBuild's pipelines are independent, so the tree is expected to be empty at the beginning of each.
+4. Manifests are expected to be machine-generated, so OSBuild has no convenience functions to support manually created manifests.
```

Comment:
Let's have this be "OSBUild's manifests..." so it's consistent with all the other lines starting with "OSBuild's".

Reply:
Or remove them all? Seems redundant since this is the osbuild document, and `stages`, `pipelines`, and `manifests` are descriptive.

Reply:
Maybe it would help to clarify this even more. The stages are a part of osbuild, the pipelines and manifests do not.
We could even link to relevant code, e.g. the stages directory or the gen-manifest script.

Reply:
@supakeen I've tried to make this more consistent, let me know if that works for you!